security/compliance_operator/compliance-operator-installation.adoc

```diff
@@ -0,0 +1,73 @@
+[id="compliance-operator-installation"]
+= Installing the Compliance Operator
+include::modules/common-attributes.adoc[]
+:context: compliance-advanced
+
+toc::[]
+
+
+Before you can actually use the operator, you need to make sure it is deployed in the cluster. Depending on your needs, you might want to deploy the upstream released packages or directly from the source.
+
+First, become kubeadmin, either with oc login or by exporting KUBECONFIG.
+
+Deploying upstream packages
+Deploying from package would deploy the latest released upstream version.
+
+First, create the CatalogSource and optionally verify it's been created successfuly:
+
+$ oc create -f deploy/olm-catalog/catalog-source.yaml
```

Comment:
for installing the supported release package, we don need to install a catalog source; we can use the Red Hat operators source which comes pre-installed in the cluster.